tests/openadapt/test_scrub.py

```diff
@@ -19,16 +21,100 @@ def _hex_to_rgb(hex_color: int) -> tuple[int, int, int]:
         tuple[int, int, int]: RGB values.
     """
     assert 0x000000 <= hex_color <= 0xFFFFFF
-    b = (hex_color >> 16) & 0xFF
-    g = (hex_color >> 8) & 0xFF
-    r = hex_color & 0xFF
-    return r, g, b
+    black = (hex_color >> 16) & 0xFF
+    green = (hex_color >> 8) & 0xFF
+    red = hex_color & 0xFF
+    return red, green, black
+
+
+def _enhance_image_for_ocr(input_image_path: str, enhanced_image_path: str) -> None:
```

Comment:
Enhance image properties for getting best results from TesseractOCR. See reference links